e2e/utils.go

```diff
@@ -30,6 +30,21 @@ const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
 //nolint:unused
 var seededRand = rand.New(rand.NewSource(time.Now().UnixNano()))
 
+// BuildArtifactName builds the artifact name from the different coordinates for the artifact
+func BuildArtifactName(artifact string, version string, OS string, arch string, extension string, isDocker bool) string {
+	dockerString := ""
+	if isDocker {
+		dockerString = ".docker"
+	}
+
+	artifactName := fmt.Sprintf("%s-%s-%s-%s%s.%s", artifact, version, OS, arch, dockerString, extension)
+	if extension == "deb" || extension == "rpm" {
```

Comment:
Maybe lowercase the string before the comparison, just to be safe?

Reply:
Resolved in https://github.com/elastic/e2e-testing/pull/689/commits/4bee7c1b7e04e715b85379e0476dc633434d6bf9